app/helpers/application_helper/button/embedded_workflow.rb

```diff
@@ -0,0 +1,11 @@
+class ApplicationHelper::Button::EmbeddedWorkflow < ApplicationHelper::Button::Basic
+  def disabled?
+    if !MiqRegion.my_region.role_active?('embedded_workflow') || ManageIQ::Providers::Workflows::Provider.count.zero?
```

Comment:
@Fryguy Does this role exist? I see this role:
<img src="https://github.com/ManageIQ/manageiq-ui-classic/assets/64800041/40df5175-d820-4e33-8260-eb5465776dc6" width="200">
but I can't enable it.

Reply:
I think we are just reusing embedded ansible ones, but I'm not sure how it should work in this tree.  @agrare?

Reply:
@DavidResende0 this line is talking about roles but what you've shown there are product features they aren't the same.
I don't think we have to have this line at all since unlike embedded_ansible there isn't a role it is always active.